src/FullBuild/IO.fs

```diff
@@ -14,14 +14,23 @@
 
 module IO
 
+open System
+
 type Result =
     { Code: int
       Info : string
       Out: string list
       Error: string list }
 
 let private resultToError execResult =
-    if execResult.Code <> 0 then (sprintf "Operation '%s' failed with error %d" execResult.Info execResult.Code) |> Some
+    if execResult.Code <> 0 then 
+        let prefix p = function [] -> [] | l -> p :: l
+
+        sprintf "Operation '%s' failed with error %d" execResult.Info execResult.Code 
```

Comment:
Would be better to always write out and error.
Just create a concatenated list of strings and get rid of prefix is not that useful.

Reply:
It just add the prefix to show which part of the text is what. And just doesn't bloat output when there is no Out or Error.

Reply:
prefix is just unreadable - get rid of this please and just ensure output is deterministic.

Reply:
Removed